### PR TITLE
Cdn upgrade Apiversion to 2024-02-01

### PR DESCRIPTION
**Pull request by Ptnan7**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by jsquire — DISMISSED** [review]
@Ptnan7 :

Your GitHub account lacks the necessary permissions to this repository and public organization memberships for this PR to move forward.  Please review the Azure SDK onboarding documentation and use the onboarding Teams channel, for assistance.

- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -668,7 +668,7 @@
 # ServiceOwners:                                                   @bastionsuppgithub
 
 # ServiceLabel: %Network - CDN
-# ServiceOwners:                                                   @cdnfdsuppgithub
+# ServiceOwners:                                                   @cdnfdsuppgithub @ptnan7 @t-bzhan
```
Comment (jsquire) [inline .github/CODEOWNERS:671]: If these are for the management library, we'll want to move it down to the management section near the bottom of the file and add "%Mgmt" to the label list.   Otherwise, this will only apply to data-plane libraries as the catch-all will triage to Arthur.  

Please see the giant comment block at the beginning of the management section for details.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -928,6 +927,8 @@
 # ServiceLabel: %Mgmt
 # AzureSdkOwners:                                                  @ArthurMa1978
 
+/sdk/cdn/                                                          @ptnan7
```
Comment (jsquire) [inline .github/CODEOWNERS:930]: Please follow the form of the other entries here.   Restrict this to the fully-qualified path for the resource manager package, add a PR label decorator to the path, add a service label + service owners block.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -669,7 +669,6 @@
 
 # ServiceLabel: %Network - CDN
```
Comment (jsquire) [inline .github/CODEOWNERS:670]: If this is intended to be the management library ownership, it will need to move into the management section around L930 and also have a `%Mgmt` entry for the `ServiceLabel`.    As this exists today, it will be treated as a data plane library label for triage.

**Comment by jsquire** [comment]
> Your GitHub account lacks the necessary permissions to this repository and public organization memberships for this PR to move forward. Please review the Azure SDK onboarding documentation and use the onboarding Teams channel, for assistance.

Account issues confirmed to be resolved.   Keeping changes requested until CODEOWNERS feedback is addressed.

**Comment by live1206** [comment]
@Ptnan7 Please update the changelog with newly added features and also the expecting release date.

**Comment by live1206** [comment]
@jsquire Could you help take another look at this PR? Thanks

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -933,6 +930,12 @@
 # ServiceLabel: %Mgmt
 # AzureSdkOwners:                                                  @ArthurMa1978
 
+# ServiceLabel: %Network - CDN %Mgmt
```
Comment (jsquire) [inline .github/CODEOWNERS:933]: Move to after the PR section, please, so that this is consistent with the other entries in the file.

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -933,6 +930,12 @@
...
+# ServiceOwners:                                                   @cdnfdsuppgithub
```
Comment (jsquire) [inline .github/CODEOWNERS:934]: The account `cdnfdsuppgithub` does not have the correct account memberships/permissions to be a CODEOWNERS contact.

**Comment by jsquire** [comment]
> @jsquire Could you help take another look at this PR? Thanks

@live1206 : CodeOwners still has unaddressed comments.

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -933,6 +930,12 @@
 # ServiceLabel: %Mgmt
 # AzureSdkOwners:                                                  @ArthurMa1978
 
+# PRLabel: %Network - CDN
```
Comment (jsquire) [inline .github/CODEOWNERS:933]: Please move this section to L951 to preserve alphabetical ordering.

**Comment by live1206** [comment]
> @Ptnan7 Please update the changelog with newly added features and also the expecting release date.

@Ptnan7 This is not resolved.

**Review by jsquire — DISMISSED** [review]

File: sdk/frontdoor/Azure.ResourceManager.FrontDoor/CHANGELOG.md [hunk 7]
```diff
@@ -1,10 +1,11 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.3.0 (2024-04-23)
```
Comment (live1206) [inline sdk/frontdoor/Azure.ResourceManager.FrontDoor/CHANGELOG.md:3]: revert this change, this PR is for CDN, you should update this in the PR for front door

File: sdk/cdn/Azure.ResourceManager.Cdn/CHANGELOG.md [hunk 8]
```diff
@@ -1,11 +1,11 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.3.0 (2024-04-23)
 
 ### Features Added
 
-
 - Enable the new model serialization by using the System.ClientModel, refer this [document](https://aka.ms/azsdk/net/mrw) for more details.
+- Update api version to 2024-02-01
```
Comment (live1206) [inline sdk/cdn/Azure.ResourceManager.Cdn/CHANGELOG.md:8]: what new features have been added in the new API version? Could you add more details?

File: sdk/cdn/Azure.ResourceManager.Cdn/CHANGELOG.md [hunk 9]
```diff
@@ -1,11 +1,11 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.3.0 (2024-04-23)
```
Comment (live1206) [inline sdk/cdn/Azure.ResourceManager.Cdn/CHANGELOG.md:3]: Since you have updated the version in changelog, you need to update the version in [Azure.ResourceManager.Cdn.csproj](https://github.com/Azure/azure-sdk-for-net/blob/696fee322caab8080ccd08fbef3458aa6f28672a/sdk/cdn/Azure.ResourceManager.Cdn/src/Azure.ResourceManager.Cdn.csproj#L3) as well

File: sdk/cdn/Azure.ResourceManager.Cdn/CHANGELOG.md [hunk 10]
```diff
@@ -1,11 +1,11 @@
...
 
 ### Features Added
 
-
 - Enable the new model serialization by using the System.ClientModel, refer this [document](https://aka.ms/azsdk/net/mrw) for more details.
+- Update api version to 2024-02-01, add support for canMigrate and Migrate API, add log-scrubbing support for profile, add new customization for CDN Endpoint.
```
Comment (live1206) [inline sdk/cdn/Azure.ResourceManager.Cdn/CHANGELOG.md:8]: consider to put each new feature in a new item

**Review by live1206 — APPROVED** [review]

**Review by ArthurMa1978 — APPROVED** [review]

**Review by ArthurMa1978 — APPROVED** [review]
